src/core/on-chain/index.ts

```diff
@@ -306,17 +329,21 @@ export const OnChainMixin = (superclass) =>
 
         const lockArgs = { logger: onchainLogger, lock }
         const result = await LockService().extendLock(lockArgs, async () => {
-          try {
-            ;({ id } = await sendToChainAddress({
-              address,
-              lnd,
-              tokens: amountToSend,
-              utxo_confirmations: 0,
-              target_confirmations: targetConfs,
-            }))
-          } catch (err) {
+          const payToAddress = () => {
```

Comment:
I don't understand the purpose of this variable. Its only being referenced in 1 place and may as well be inline, or?

Reply:
is to avoid to have `onchainLogger.error` twice. This willl be remove in the final PR